src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission.UnitTests/SubmissionEventsProcessorTests/WhenProcessing.cs

```diff
@@ -148,13 +159,15 @@ public void ThenItShouldWriteAnEventForEachIlrDeliveryThatHasChanged()
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ProgrammeType == null)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].FrameworkCode == null)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].PathwayCode == null)));
-            _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ActualStartDate == null)));
+            _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ActualStartDate == updatedSubmissionChanged.ActualStartDate)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].PlannedEndDate == updatedSubmissionChanged.PlannedEndDate)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].ActualEndDate == null)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].OnProgrammeTotalPrice == updatedSubmissionChanged.OnProgrammeTotalPrice)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].CompletionTotalPrice == updatedSubmissionChanged.CompletionTotalPrice)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[1].NiNumber == null)));
             _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[0].EPAOrgId == ilrForFirstSubmission.EPAOrgId)));
+            _mediator.Verify(m => m.Send(It.Is<WriteSubmissionEventCommand>(c => c.Events[0].GivenNames == ilrForFirstSubmission.GivenNames)));
```

Comment:
The last 3 lines should be checking c.Events[1]

Reply:
😡 Damn it!  Fixing up now.  👍

Reply:
Fixed below.